integration_tests/examples/easy_test.rs

```diff
@@ -0,0 +1,202 @@
+    examples::{self, run_example, TestSample},
+    ffmpeg::{start_ffmpeg_receive, start_ffmpeg_send_sample},
+};
+
+const VIDEO_RESOLUTION: Resolution = Resolution {
+    width: 1280,
+    height: 720,
+};
+
+const IP: &str = "127.0.0.1";
+const INPUT_1_PORT: u16 = 8002;
+const INPUT_2_PORT: u16 = 8004;
+const INPUT_3_PORT: u16 = 8006;
+const INPUT_4_PORT: u16 = 8008;
+const INPUT_5_PORT: u16 = 8014;
+const INPUT_6_PORT: u16 = 8016;
+const INPUT_7_PORT: u16 = 8018;
+const OUTPUT_VIDEO_PORT: u16 = 8010;
+const OUTPUT_AUDIO_PORT: u16 = 8012;
+
+fn main() {
+    run_example(start_example_client_code);
+}
+
+fn start_example_client_code() -> Result<()> {
+    info!("[example] Start listening on output port.");
+    start_ffmpeg_receive(Some(OUTPUT_VIDEO_PORT), Some(OUTPUT_AUDIO_PORT))?;
+
+    info!("[example] Send register input request.");
+    examples::post(
+        "input/input_1/register",
+        &json!({
+            "type": "rtp_stream",
+            "port": INPUT_1_PORT,
+            "video": {
+                "decoder": "ffmpeg_h264"
+            },
+        }),
+    )?;
+
+    info!("[example] Send register input request.");
+    examples::post(
+        "input/input_2/register",
+        &json!({
+            "type": "rtp_stream",
+            "port": INPUT_2_PORT,
+            "audio": {
+                "decoder": "opus"
+            },
+        }),
+    )?;
+
+    info!("[example] Send register input request.");
+    examples::post(
+        "input/input_3/register",
+        &json!({
+            "type": "rtp_stream",
+            "port": INPUT_3_PORT,
+            "video": {
+                "decoder": "ffmpeg_h264"
+            },
+        }),
+    )?;
+
+    info!("[example] Send register input request.");
+    examples::post(
+        "input/input_4/register",
+        &json!({
+            "type": "rtp_stream",
+            "port": INPUT_4_PORT,
+            "audio": {
+                "decoder": "opus"
+            },
+        }),
+    )?;
+
+    info!("[example] Send register input request.");
+    examples::post(
+        "input/input_5/register",
+        &json!({
+            "type": "rtp_stream",
+            "port": INPUT_5_PORT,
+            "video": {
+                "decoder": "ffmpeg_h264"
+            },
+        }),
+    )?;
+
+    info!("[example] Send register input request.");
+    examples::post(
+        "input/input_6/register",
+        &json!({
+            "type": "rtp_stream",
+            "port": INPUT_6_PORT,
+            "video": {
+                "decoder": "ffmpeg_h264"
+            },
+        }),
+    )?;
+
+    info!("[example] Send register input request.");
+    examples::post(
+        "input/input_7/register",
+        &json!({
+            "type": "rtp_stream",
+            "port": INPUT_7_PORT,
+            "video": {
+                "decoder": "ffmpeg_h264"
+            },
+        }),
+    )?;
+
+    info!("[example] Send register output request.");
+    examples::post(
```

Comment:
If you use info logs every time, consider moving them into the `examples::post` function.
You could remove those logs in every example with it.

Logging the route should be enough (e.g. `info!("Sending post request to {}", route)`).

Reply:
In general, I would just log things like
```rust
    info!("[example] Start listening on output port.");
```
in those helper functions.

With it, you could remove all those `info` calls from example files.

It's for testing only, so additional logs are not a problem IMO.